css/media.css

```diff
@@ -0,0 +1,278 @@
+@media (max-width: 768px) {
+  .header {
+    box-sizing: border-box;
+    padding: 24px 33px 0 15px;
+  }
+
+  .header__menu {
+    display: none;
+  }
+
+  .link-none {
+    display: none;
+  }
+
+  .link__img {
+    width: 35.23px;
+    height: 22.02px;
+    margin-left: auto;
+    margin-right: 0;
+  }
+
+  .car__title {
+    margin-top: 80px;
+  }
+
+  .car__subtitle {
+    position: relative;
+    bottom: 10px;
+    padding-left: 16px;
+  }
+
+  .car {
+    background-size: 109%;
+    background-position: 15% 50%;
+  }
+
+  .car__tsl {
+    width: 91%;
+    position: relative;
+    left: 10px;
+    top: 65px;
+  }
+
+  .welcome-page {
+    margin-top: 104px;
+    display: block;
+  }
+
+  .welcome-page__1 {
+    width: 100%;
+  }
+
+  .welcome-page__speed-sec {
+    font-size: 50px;
+    line-height: 61px;
+  }
+
+  .welcome-page__driver-max {
+    font-size: 50px;
+    line-height: 61px;
+  }
+
+  .welcome-page__speed-car {
+    width: 120%;
+    padding: 18px 0 0 10px;
+  }
+
+  .discrip-mob {
+    margin-top: 2px;
+    margin-left: 38px;
+  }
+
+  .discrip-sek {
+    padding-left: 5px;
+    margin-bottom: 10px;
+    position: relative;
+    bottom: 8px;
+  }
+
+  .welcome-page__driver {
+    margin-right: 30px;
+  }
+
+  .welcome-page__drive_img {
+    margin-right: 14px;
+    margin-top: 10px;
+  }
+
+  .km {
+    position: relative;
+    bottom: 7px;
+    left: 6px;
+  }
+
+  .welcome-page__driver-go {
+    margin-top: -2px;
+  }
+
+  .welcome-page__2 {
+    margin-top: 39px;
+  }
+
+  .welcome-page__test-drive {
+    font-size: 16px;
+    line-height: 20px;
+    color: #e51937;
+    margin-left: 15px;
+    margin-top: 3px;
+  }
+
+  .welcome-page__footer-btn {
+    font-size: 16px;
+    line-height: 20px;
+    color: #fff;
+  }
+
+  .welcome-page__footer {
+    background: #e51937;
+    border: 0;
+    width: 220.84px;
+    height: 55.21px;
+    margin-right: 54px;
+  }
+
+  .welcome-page__button {
+    margin-left: 64px;
+    margin-top: 10px;
+  }
+
+  .arrow {
+    margin-top: 63px;
+  }
+}
+
+@media (max-width: 375px) {
+  .header {
+    margin: 5px 21px 0 2px;
+    padding: 0;
+  }
+
+  .header__img {
+    width: 84%;
+  }
+
+  .link__img {
+    width: 45%;
+    height: 50%;
+  }
+
+  .car__title {
+    font-size: 50px;
+    line-height: 61px;
+    margin-top: 30px;
+  }
+
+  .car__subtitle {
+    font-size: 10px;
+    line-height: 12px;
+    position: relative;
+    bottom: 2px;
+    padding-left: 2px;
+  }
+
+  .car {
+    background-size: 113%;
+    background-position: 30% 70%;
+  }
+
+  .car__tsl {
+    width: 95%;
+    position: relative;
+    left: 1px;
+    top: 40px;
+  }
+
+  .welcome-page__speed-sec {
+    font-size: 27px;
+    line-height: 33px;
+  }
+
+  span {
+    font-size: 15px;
+    line-height: 18px;
+  }
+
+  .welcome-page__speed-km {
+    font-size: 10px;
+    line-height: 12px;
+  }
+  
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/teslaproject/pullRequest?prid=5566861)